lib/rules/web/admin_console/4.19/goto.xyaml

```diff
@@ -225,7 +225,7 @@ goto_deploy_image_page:
   url: deploy-image/ns/<project_name>?preselected-ns=<project_name>
   element:
     selector:
-      xpath: //div[contains(@class,'pf-v5-c-form__helper-text') and contains(., "Deploy")]
+      xpath: //div[contains(@class,'-c-form__helper-text') and contains(., "Deploy")]
```

Comment:
Can we just use `//div[contains(., "Deploy")]`? It looks like this is just a element to ensure page is loaded

Reply:
I guess not
![image](https://github.com/user-attachments/assets/708f3989-20f2-4506-a563-9b06dd6ae28c)